### ISSUE TITLE
New Feature : add Prelist of pipelines

### ISSUE THREAD

**Issue by arita37** [rhiever/tpot#296]
When we have worked on a class of datasetb (larget size, 1Go), we have an idea
of the optimal pipeline, and we just want to investigate the optimal parameters without computing the whole in genetic programm. It would reduce substantially the computation time.

For example:
  We can provide a list of pipeline with algorithm names ['LinearSVC', 'RF' ]
  We can provide a list of hyper-parameter for each algorithm to search for...

Main goal is to limit the search to those parameters.

**Comment by rhiever** [comment]
This can currently be "hacked" into TPOT by modifying the [population instantiation procedure](https://github.com/rhiever/tpot/blob/master/tpot/base.py#L296), but it could be a nice feature to be able to do that through the regular TPOT interface. We'd have to put a lot of thought into how that would look. I'll file this as an enhancement request for now.

**Comment by rhiever** [comment]
@teaearlgraycold is taking on this issue. He will be extending the `config_dict` parameter to support custom pipeline seeds.

**Comment by hanshupe** [comment]
> This can currently be "hacked" into TPOT by modifying the [population instantiation procedure](https://github.com/rhiever/tpot/blob/master/tpot/base.py#L296), but it could be a nice feature to be able to do that through the regular TPOT interface. We'd have to put a lot of thought into how that would look. I'll file this as an enhancement request for now.

I would like to use the "hack", but I think the link is outdated. Could someone please update the Line number, where I can add the initial solutions? Thx!

**Comment by weixuanfu** [comment]
I think it was refereed to [this line](https://github.com/EpistasisLab/tpot/blob/v0.11.5/tpot/base.py#L670)

**Comment by hanshupe** [comment]
> I think it was refereed to [this line](https://github.com/EpistasisLab/tpot/blob/v0.11.5/tpot/base.py#L670)

Can you give me a hint how I can convert a predefined sklearn pipeline into a list of DEAP primitives that is needed for the population?

**Comment by weixuanfu** [comment]
@hanshupe we don't have a example for this conversion . But we have [a example for reverse conversion](https://github.com/EpistasisLab/tpot/blob/master/tests/tpot_tests.py#L358-L370).

**Comment by hanshupe** [comment]
Yes thanks I saw the compile method, but the conversion into the opposite direction is not so easy as I thought :)
